### PR TITLE
Add license to README and create issue templates (Closes #177)

### LINKED ISSUE #177: Issue templates and license
Please add issue templates to this repository. 
I would also suggest that you add the license to the README.md, as done by (nearly) all other CWA repositories ([link](https://github.com/corona-warn-app/cwa-app-ios#licensing) to iOS repo)

Let me know wether a PR reg. this would be welcome.

Best wishes
Tim

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR creates issue templates and adds the license to the README.md. 

---

Closes #177

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/05_other.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+name: "\U0001F4AC Other"
+about: For conceptual questions, please consider opening an issue in the documentation repository.
+labels: other
+---
+
+<!--
+Thanks for contributing to the project!
+Before opening a new issue, please make sure that we do not have any duplicates already open. 
+You can ensure this by searching the issue list for this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead.
+Also, be sure to check our readme first: https://github.com/corona-warn-app/cwa-verification-server
```
Comment (SebastianWolf-SAP) [inline .github/ISSUE_TEMPLATE/05_other.md:11]:
```suggestion
Also, be sure to check our readme first: https://github.com/corona-warn-app/cwa-ppa-server
```

File: README.md [hunk 2]
```diff
@@ -193,6 +193,16 @@ become part of its developer community.
...
+## Licensing
+
+Copyright (c) 2020-2021 SAP SE or an SAP affiliate company.
+
+Licensed under the **Apache License, Version 2.0** (the "License"); you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License from [here](./LICENSE).
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the [LICENSE](./LICENSE) for the specific language governing permissions and limitations under the License.
+
```
Comment (SebastianWolf-SAP) [inline README.md:205]:
```suggestion

Please see the [detailed licensing information](https://api.reuse.software/info/github.com/corona-warn-app/cwa-ppa-server) via the [REUSE Tool](https://reuse.software/) for more details.
```

**Review by SebastianWolf-SAP — CHANGES_REQUESTED** [review]
Thank you very much! Only two very small change requests:
- Copy/Paste issue with the link to the README
- The licensing section might also get a sentence on the detailed licensing information by the REUSE tool

**Comment by Ein-Tim** [comment]
Thanks @SebastianWolf-SAP, I committed your suggested changes.

**Review by SebastianWolf-SAP — APPROVED** [review]
That was quick! Thank you very much, approved! :)

**Review by hilmarf — APPROVED** [review]

**Pull request merged**